db/view/view.cc

```diff
@@ -1864,12 +1864,23 @@ future<> view_update_generator::mutate_MV(
     // TODO: Maybe allow users to set use_legacy_self_pairing explicitly
```

Comment:
After https://github.com/scylladb/scylladb/pull/21781 gets merged, we'll be able to use just a single topology in `get_view_natural_endpoint`, maybe we can note this down somewhere (commit message, FIXME) or update https://github.com/scylladb/scylladb/pull/21781 or this PR, depending on what gets merged first

Reply:
Added FIXME comment in v11 ([89cc967](https://github.com/scylladb/scylladb/commit/89cc967f99ce65b260bcd0b037e232fe657fd1d3))